src/Microsoft.DotNet.Wpf/src/PresentationFramework/System/Windows/Documents/WinRTSpellerInterop.cs

```diff
@@ -886,16 +897,18 @@ public int Length
         }
 
         [DebuggerDisplay("SubSegments.Count = {SubSegments.Count} TextRange = {TextRange.Start},{TextRange.Length}")]
-        private class SpellerSegment: ISpellerSegment
+        internal class SpellerSegment: ISpellerSegment
         {
             #region Constructor
 
-            public SpellerSegment(WordSegment segment, SpellChecker spellChecker, WinRTSpellerInterop owner)
+            public SpellerSegment(string sourceString, ITextRange textRange, SpellChecker spellChecker, WinRTSpellerInterop owner)
```

Comment:
NetFx needs them to enable this code pattern behind an application-compatibility quirk. Since this quirk is not being introduced to .NET Core... 

```C#
                        if (!FrameworkAppContextSwitches.DoNotAugmentWordBreakingUsingSpeller)
                        {
                            _segments = _wordBreaker.ComprehensiveGetTokens(_sentence, _spellChecker, _owner);
                        }
                        else
                        {
                            var segments = new List<SpellerSegment>();
                            foreach (var wordSegment in _wordBreaker.GetTokens(_sentence))
                            {
                            segments.Add(new SpellerSegment(wordSegment, _spellChecker, _owner));
                                segments.Add(new SpellerSegment(_sentence, wordSegment, _spellChecker, _owner));
                            }

                            _segments = segments.AsReadOnly();
                        }
```